### ISSUE TITLE
Training Dataset

### ISSUE THREAD

**Issue by YoungJoongUNC** [YapengTian/TDAN-VSR#12]
Hello.
May I ask a question regarding the training dataset?

Did you use the training set of Vimeo 90k septuplet dataset for video super-resolution  (The original training + test set (consists of 91701 sequences, which are not downsampled or downgraded by noise): zip (82GB) ) for the training without any processing (like changing the name of each file or sampling frames)?

Thank you.

**Comment by YapengTian** [comment]
Yes, I used the training set of Vimeo 90k and adopted the scripts (BI: bicubic downsampling and BD: https://github.com/yulunzhang/RDN/blob/master/RDN_IR/RDN_TestCode/RDN_SR_RGB/Prepare_TestData_HR_LR.m) provided by the authors to generate training pairs and didn't change names.

**Comment by YoungJoongUNC** [comment]
The link points to the script to generate the testing dataset. Did you generated the dataset using this code and use it as the training dataset?

**Comment by YoungJoongUNC** [comment]
I guess you were referring to the following scripts (prepare train data with BI and BD) right? 

https://github.com/yulunzhang/RDN/blob/master/RDN_TrainCode/Prepare_TrainData/Prepare_TrainData_HR_LR_BD.m

https://github.com/yulunzhang/RDN/blob/master/RDN_TrainCode/Prepare_TrainData/Prepare_TrainData_HR_LR_BI.m

May I ask if you used both methods (BI and BD) to generate train dataset? Then may I ask how did you divide the train dataset?

**Comment by YapengTian** [comment]
Yes. I followed the ToFlow paper and used their videos from the train list:https://github.com/anchen1011/toflow/blob/master/data/sep_trainlist.txt.

**Comment by YoungJoongUNC** [comment]
I think I got confused a bit. My understanding is that you only used bicubic (only BI not BD) downsampling to generate training dataset. And you modified RDN's "testing" dataset generation code (https://github.com/yulunzhang/RDN/blob/master/RDN_IR/RDN_TestCode/RDN_SR_RGB/Prepare_TestData_HR_LR.m) to generate your "training set". Would this right?

And I found that the RDN dataset generation code (which you uploaded the link) deals with non-video dataset(DIV2 dataset) while your dataset(Vimeo 90k dataset) is video dataset.
RDN's code use DIV2 dataset where high resolution images are organized like this: 0001.png, 0002.png, ..., 0800.png
But Vimeo 90k dataset is organized like this: 00001/0001~1000/im1.png~im7.png.

May I ask how did you modify RDN's dataset generation code to generate the train dataset consists of the Vimeo 90k dataset ? May I ask if you could make the dataset generation code public or give more detailed descriptions on how to generate train and test dataset?

**Comment by YapengTian** [comment]
I used both BI and BD to generate LR frames as described in the paper. I used the scripts from  Toflow: https://github.com/anchen1011/toflow/blob/master/src/generate_testing_sample/downsample_input.m to generate LR frames. If you also want to explore  BD setting, you could use BD code from https://github.com/anchen1011/toflow/blob/master/src/generate_testing_sample/downsample_input.m.
